manubot/cite/handlers.py

```diff
@@ -157,6 +157,252 @@ def get_csl_item(self, citekey: CiteKey) -> Dict[str, Any]:
     "4dn": "manubot.cite.curie.Handler_CURIE",
     "4dn.biosource": "manubot.cite.curie.Handler_CURIE",
     "4dn.replicate": "manubot.cite.curie.Handler_CURIE",
+    "AAO": "manubot.cite.curie.Handler_CURIE",
```

Comment:
IIRC the keys here should be all lowercase, because lookup of handlers is always done with lowercase prefixes. So these capitalized handlers should never get called.